bjoern/request.c

```diff
@@ -291,8 +301,18 @@ parser_settings = {
 
 void _initialize_request_module()
 {
+    IO = PyImport_ImportModule("io");
+    if (IO == NULL) {
+	    /* PyImport_ImportModule should have exception set already */
+	    return;
+    }
+    STR_bytesio = _Unicode_FromString("BytesIO");
```

Comment:
put these constant strings into the common module and use the `_(...)` macro (if possible)

Reply:
Will move the common strings to common.